platforms/corda-ent/charts/notary/templates/pvc.yaml

```diff
@@ -19,4 +19,4 @@ spec:
   storageClassName: {{ .Values.storage.name }}
   resources:
     requests:
-      storage: 64Mi
+      storage: 512Mi
```

Comment:
Storage size is hardcoded. Move this to `values.yaml` and add to `notary.tpl` file.